.install/install_rust.sh

```diff
@@ -19,9 +19,12 @@ rustup update
 #   is missing one of the components we need.
 # llvm-tools-preview:
 #   Required by `cargo-llvm-cov` for test coverage
+# miri:
+#   Required to run `cargo miri test` for UB detection
 # rust-src:
 #   Required to build RedisJSON with address sanitizer
 rustup toolchain install nightly \
     --allow-downgrade \
     --component llvm-tools-preview \
+    --component miri \
```

Comment:
here is a general question
since nightly is only for sanitizer, why don't we move those to test deps installation? I assume we don't use nightly features on production code